client/src/components/Login.js

```diff
@@ -3,11 +3,11 @@ import { Button, Form } from "semantic-ui-react";
 import { Redirect } from "react-router-dom";
 import { signApi } from "../api/auth";
 
-const ApplicantLogin = () => {
+const Login = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [error, setError] = useState(null);
-  const [success, setSuccess] = useState(false);
+  const [successRoute, setSuccessRoute] = useState(false);
```

Comment:
Default state should be an empty string instead of false and you can rename successRoute to `redirectRoute`.
E.g `const [redirectRoute, setRedirectRoute] = useState("");`

Reply:
Done. Thanks.